src/main/java/com/thirdcc/webapp/web/rest/EventAttendeeResource.java

```diff
@@ -114,6 +114,14 @@ public EventAttendeeResource(EventAttendeeService eventAttendeeService) {
         return ResponseUtil.wrapOrNotFound(eventAttendeeDTO);
     }
 
+    @GetMapping("/event-attendees/event/{eventId}")
+    public ResponseEntity<List<EventAttendeeDTO>> getAllEventAttendeeByEventId(Pageable pageable, @PathVariable Long eventId, @RequestParam MultiValueMap<String, String> queryParams, UriComponentsBuilder uriBuilder) {
```

Comment:
for this one is it better to add the @preAuthorize annotation to check whether the person trying to access this method is admin or crew?